numpy/distutils/misc_util.py

```diff
@@ -1912,11 +1909,11 @@ def get_version(self, version_file=None, version_variable=None):
         for f in files:
             fn = njoin(self.local_path, f)
             if os.path.isfile(fn):
-                info = (open(fn), fn, ('.py', 'U', 1))
+                info = (fn, ('.py', 'U', 1))
                 name = os.path.splitext(os.path.basename(fn))[0]
                 n = dot_join(self.name, name)
                 try:
-                    version_module = imp.load_module('_'.join(n.split('.')),*info)
+                    version_module = import_module('_'.join(n.split('.')), *info)
```

Comment:
Agreed, that old form of `info` is a bit confusing (and it does not seem to be used anywhere else)